src/js/containers/covid19/recipient/RecipientTableContainer.jsx

```diff
@@ -0,0 +1,347 @@
+                </Link>
+            );
+        }
+        if (activeTab === 'loans') {
+            return [
+                link,
+                rowData.obligation,
+                rowData.outlay,
+                rowData.faceValueOfLoan,
+                rowData.awardCount
+            ];
+        }
+        return [
+            link,
+            rowData.obligation,
+            rowData.outlay,
+            rowData.awardCount
+        ];
+    })
+);
+
+const RecipientTableContainer = ({ activeTab, prevActiveTab, scrollIntoView }) => {
+    const [currentPage, changeCurrentPage] = useState(1);
+    const [prevPageSize, pageSize, changePageSize] = useStateWithPrevious(10);
+    const [totalItems, setTotalItems] = useState(0);
+    const [results, setResults] = useState([]);
+    const [parsedRows, setParsedRows] = useState([]); // finalized rows (including Unlinked Data)
+    const [resultTotal, setResultTotal] = useState(0);
+    const [loading, setLoading] = useState(true);
+    const [error, setError] = useState(false);
+    const [prevSort, sort, setSort] = useStateWithPrevious('obligation');
+    const [prevOrder, order, setOrder] = useStateWithPrevious('desc');
+    const [prevQuery, query, setQuery] = useStateWithPrevious('');
+    const tableRef = useRef(null);
+    const tableWrapperRef = useRef(null);
+    const errorOrLoadingWrapperRef = useRef(null);
+    const request = useRef(null);
+    const [unlinkedDataClass, setUnlinkedDataClass] = useState(false);
+    const { recipientTotals, defcParams } = useSelector((state) => state.covid19);
+
+    const updateSort = (field, direction) => {
+        setSort(field);
+        setOrder(direction);
+    };
+
+    const addUnlinkedData = (rows = results, totals = resultTotal, totalRecipient = recipientTotals) => {
+        // add unlinked data if activeTab is all
+        if (activeTab === 'all' && !query && Object.keys(totalRecipient).length > 0) {
+            const unlinkedData = calculateUnlinkedTotals(totalRecipient, totals);
+            setUnlinkedDataClass(true);
+            const unlinkedName = (
+                <div className="unlinked-data">
+                    Unknown Recipient (Unlinked Data)
+                </div>
+            );
+            const rowData = Object.create(BaseSpendingByRecipientRow);
+
+            rowData.populate({
+                obligation: unlinkedData.obligation,
+                outlay: unlinkedData.outlay,
+                award_count: unlinkedData.award_count
+            });
+
+            const rowsWithUnlinked = rows
+                .concat([[
+                    unlinkedName,
+                    rowData.obligation,
+                    rowData.outlay,
+                    rowData.awardCount
+                ]]);
+            setParsedRows(rowsWithUnlinked);
+        }
+        else {
+            setParsedRows(rows);
+        }
+    };
+
+    const fetchSpendingByRecipientCallback = useCallback(() => {
```

Comment:
@ebdabbs I see this warning in ESlint:

React Hook useCallback does nothing when called with only one argument. Did you forget to pass an array of dependencies?

Reply:
But now I see that this was there before.

Reply:
Removing `useCallback` since I don't think it's necessary here.